src/main/kotlin/com/gamingbacklog/api/gamingbacklogapi/models/User.kt

```diff
@@ -0,0 +1,14 @@
+package com.gamingbacklog.api.gamingbacklogapi.models
+
+import org.bson.types.ObjectId
+import org.springframework.data.annotation.Id
+import org.springframework.data.mongodb.core.mapping.Document
+
+@Document("users")
+data class User(
+  @Id
+  val id: String = ObjectId.get().toString(),
+  var username: String,
```

Comment:
At work we recently went through a huge migration to unify emails and usernames, so I have to ask:

Do we want to have separate usernames and emails like github? Maybe allow users multiple email addresses? Or do we want an email address that uniquely identifies the user, and a display name that they can customize (but won't be able to log in with - like Goodreads, I think). I definitely lean towards the display name model, but just wanted to clarify what our goal was here

Reply:
That's a good question. We can discuss this further when we solidify design, but I'm inclined to agree. I feel like these days when I log into a site, I usually do the google automatic login magic thing, so I do feel like email as a requirement to login and username as just a display name that's changeable is good. To make this clear, I'll change the keyword "username" to "displayName"